va-gov/pages/gi-bill-comparison-tool/index.md

```diff
@@ -6,6 +6,6 @@ layout: page-react.html
 entryname: gi
 collection: education
 spoke: More Resources
-order: 4
+order: 1
```

Comment:
There was a duplicate content page for this (just a stub for sidebars), which had the right `order` prop, but this one is in the right location.